### PR TITLE
fix for bootstrap properties code completion replacing existing line

### PR DESCRIPTION
**Pull request by arunvenmany-ibm**

### CODE REVIEW

**Comment by arunvenmany-ibm** [comment]
https://github.com/user-attachments/assets/76384561-4461-4a24-b5a6-59508ac789cd

File: liberty-ls/src/main/java/io/openliberty/tools/langserver/model/propertiesfile/PropertiesKeyInstance.java [hunk 1]
```diff
@@ -16,6 +16,7 @@
 import org.eclipse.lsp4j.CompletionItem;
 import org.eclipse.lsp4j.CompletionItemKind;
 import org.eclipse.lsp4j.Hover;
+import org.eclipse.lsp4j.InsertReplaceEdit;
```
Comment (cherylking) [inline liberty-ls/src/main/java/io/openliberty/tools/langserver/model/propertiesfile/PropertiesKeyInstance.java:19]: Update copyright to include 2024.

**Review by cherylking — CHANGES_REQUESTED** [review]
The change looks good. Just one question...why did I see a diagnostic flagged when you selected the bootstrap property `com.ibm.ws.logging.console.source` in the video (at 18 seconds)?

**Comment by arunvenmany-ibm** [comment]
> The change looks good. Just one question...why did I see a diagnostic flagged when you selected the bootstrap property `com.ibm.ws.logging.console.source` in the video (at 18 seconds)?

Thats because we have not set the property value. once set value for the property, diagnostic disappeared

image
image

Once some value is set, diagnostic is disappeared
image

File: liberty-ls/src/main/java/io/openliberty/tools/langserver/model/propertiesfile/PropertiesKeyInstance.java (hunk 1 above)
Comment (arunvenmany-ibm) [reply liberty-ls/src/main/java/io/openliberty/tools/langserver/model/propertiesfile/PropertiesKeyInstance.java:19]: Done

**Review by cherylking — APPROVED** [review]

**Pull request merged**